src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,9 @@
-const booleanToWord = (boolean) => {};
+const booleanToWord = (boolean) => {
+    if (boolean == true) {
+        return 'Yes';
+    } else {
+return 'No';
+    }
+};
```

Comment:
This works, but it could be made more concise using a ternary operator:
```
  return (boolean === true) ? "yes" : "no";
```
To go a little bit further, this could all be put on one line like this:
```
const booleanToWord = (boolean) => (boolean === true ? "yes" : "no");
```
This is the exact same as above, as when putting a function expression on one line like this you no longer need to use the `return` keyword as it is automatically implied